h_transport_materials/material.py

```diff
@@ -147,3 +147,14 @@ class MoltenSalt(Material):
 
 ALUMINA = Compound("alumina")
 EUROFER = Steel("eurofer_97")
+
+APMT = Alloy("apmt")
+T35Y = Alloy("t35y")
+T54Y = Alloy("t54y")
+
+THERMACORE = Alloy("thermacore")
+THERMACORE_OXIDIZED = Alloy("thermacore_oxidized")
+
+FE22CR5AL = Alloy("fe22cr5al")
+
+OXIDIZED_1605 = Alloy("oxidized_1605")
```

Comment:
Are these alloys steel? If so the material objects could be instances of `Steel` instead of instances of `Alloy`

Reply:
Moreover, if all these materials are strains of FeCrAl alloy, then we can create a `FeCrAl` c lass/family (maybe inheriting from the `Steel` class) and these materials would be instances of `FeCrAl`